.changeset/gold-windows-fly.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/lit': patch
```

Comment:
It is technically a breaking change due to the changes in the available dom shim. would it make sense to do a new major?